tests/test_psf.py

```diff
@@ -0,0 +1,57 @@
+"""
+Unit tests for PSF-related code.
+"""
+import os
+import glob
+import unittest
+import desc.imsim
+
+
+def are_psfs_equal(psf1, psf2):
+    """
+    Test that two PSFs are equal.  For PSFs implemented in
+    sims_GalSimInterface, it is sufficient to compare the
+    ._cached_psf attributes.  For the AtmosphericPSF, only certain
+    attributes are meaningful to compare. See issue #117.
+    """
+    if type(psf1) != type(psf2):
+        return False
+    if not isinstance(psf1, desc.imsim.atmPSF.AtmosphericPSF):
+        # Compare cached galsim objects.
+        return psf1._cached_psf == psf2._cached_psf
+    # See issue #117 for an explanation of these comparisons:
+    return (psf1.atm[:-1] == psf2.atm[:-1]) and (psf1.aper == psf2.aper)
```

Comment:
Rather than this awkward comparison, I think we should write an `__eq__` for `AtmosphericPSF` (and whatever classes also need an `__eq__` downstream).

Reply:
I pushed an update.  

I think this is ready to go now.  I'll rebase branch r0_500_fsolve once this is merged.

Reply:
Thanks, Josh.  Now that `AtmosphericPSF` has `__eq__`, I will issue a PR at sims_GalSimInterface to provide that method for the other PSF classes so that we can clean up that test program.